src/dmarc/index.ts

```diff
@@ -0,0 +1,112 @@
+import * as route53 from 'aws-cdk-lib/aws-route53';
+import { Construct } from 'constructs';
+import { EmailReceiver, EmailReceiverProps } from '../email-receiver/receiver';
+
+export interface DmarcReporterProps {
+  /**
+   * The Route 53 hosted zone to create the DMARC record in.
+   */
+  hostedZone: route53.IHostedZone;
+  /**
+   * The email address to send DMARC reports to.
+   * This email address must be verified in SES.
+   */
+  emailAddress: string;
```

Comment:
Can we default to `dmarc-reports` @ the domain name of the provided hosted zone?